src/Symfony/Component/AssetMapper/Command/ImportMapUpdateCommand.php

```diff
@@ -33,21 +34,37 @@ public function __construct(
     protected function configure(): void
     {
         $this
+            ->addArgument('packages', InputArgument::IS_ARRAY | InputArgument::OPTIONAL, 'List of packages\' names')
             ->setHelp(<<<'EOT'
 The <info>%command.name%</info> command will update all from the 3rd part packages
 in <comment>importmap.php</comment> to their latest version, including downloaded packages.
 
    <info>php %command.full_name%</info>
+
+Or specific packages only:
+
+    <info>php %command.full_name% <packages></info>
 EOT
-            );
+            )
+        ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $packages = $input->getArgument('packages');
+
         $io = new SymfonyStyle($input, $output);
-        $this->importMapManager->update();
+        $this->importMapManager->update($packages);
```

Comment:
Idea:

```suggestion
        $updatedPackages = this->importMapManager->update($packages);
```

Then, below, `\count($updatedPackages)`. That would probably show psalm that the return value IS used.

Reply:
Done :+1: